readthedocs_theme/templates/includes/topnav.html

```diff
@@ -118,30 +139,147 @@
             {{ menu_main() }}
 
             <div class="right menu">
-              <div class="ui wide dropdown item" data-module="dropdown" data-action="select" data-display-type="block">
-                <span class="text">Help</span>
-                <i class="fad fa-caret-down icon"></i>
-                <div class="menu">
-                  {{ menu_help() }}
+
+              <div class="item">
+                {# TODO: Add support page to marketing website #}
+                <div class="ui wide dropdown item" data-module="dropdown" data-action="select" data-display-type="block">
+                  <a aria-label="Help" class="ui circular basic icon button">
+                    <i class="fa-solid fa-message-question icon"></i>
+                  </a>
+                  <div class="menu">
+                    {{ menu_help() }}
+                  </div>
                 </div>
               </div>
-              <div class="item">
 
-                <div class="ui fluid dropdown primary button" data-module="dropdown" data-action="select" data-display-type="block">
-                  <i class="fad fa-right-to-bracket icon"></i>
-                  <span class="text">Log in</span>
+              <div class="item">
+                <div class="ui fluid dropdown" data-module="dropdown" data-action="select" data-display-type="block">
+                  <a class="ui" href="/accounts/login/">
+                    Log in
+                  </a>
                   <div class="menu">
                     {{ menu_log_in() }}
-                  </div>
+  -                </div>
                 </div>
+              </div>
 
+              <div class="item">
+                <a class="ui button"
+                data-analytics="signup-modal"
+                 onclick="jQuery('#signup-modal').modal('show');">
+                  Sign up
+                </a>
               </div>
-            </div>
 
+            </div>
           </div>
         </div>
       {% endblock topnav_menu %}
 
     </div>
   </div>
 </div>
+
+
+{% block signup_modal %}
```

Comment:
This modal will be defined in two places with this, again here:
https://github.com/readthedocs/website/blob/8095be9acdda237df82da32f39c23b292dc0bb37/content/pages/homepage.html#L42-L44

We should remove the content page source version, as we will already be defining this on every single page.